packages/inter-protocol/test/psm/test-psm.js

```diff
@@ -209,29 +213,49 @@ async function makePsmDriver(t, customTerms) {
       return feePayoutAmount;
     },
 
-    /** @param {Amount<'nat'>} giveAnchor */
-    async swapAnchorForStable(giveAnchor) {
+    /**
+     * @param {Amount<'nat'>} giveAnchor
+     * @param {Amount<'nat'>} [wantStable]
+     * @param {*} [out]
```

Comment:
yay for typing. this param would benefit from more docs:
```
```suggestion
     * @param {{seat: UserSeat, offerResult: *}} [out] will be filled
```

Reply:
But I find the need for this suspicious. This function is returning payouts to abstract away seat and offerResult… but here we are needing to punch through. Will contract users need to also? If so we should just change the return type to hide less: `{seat, offerResult, payouts}`.

Or would it be sufficient to have `{numWantsSatisfied, payouts}`?

Reply:
Yes, I was too lazy to change the other call sites. Returning just the seat might be most straightforward.

Reply:
I refactored the driver to have additional methods that expose the seat.